lib/method-map.js

```diff
@@ -0,0 +1,117 @@
+// https://github.com/microsoft/WinAppDriver/blob/master/Docs/SupportedAPIs.md
+
+export const newMethodMap = /** @type {const} */ ({
+  '/session/:sessionId/appium/start_recording_screen': {
+    POST: {
+      command: 'startRecordingScreen',
+      payloadParams: { optional: ['options'] }
+    }
+  },
+  '/session/:sessionId/appium/stop_recording_screen': {
+    POST: {
+      command: 'stopRecordingScreen',
+      payloadParams: { optional: ['options'] }
+    }
+  },
+  '/session/:sessionId/appium/device/push_file': {
```

Comment:
pushFile, pullFile, and pullFolder have not been deprecated in appium so there's probably no need to override them here?

Reply:
these are not present in the standard, so I would rather keep them.